addons/project/static/tests/tours/project_sharing_tour.js

```diff
@@ -224,7 +225,14 @@ registry.category("web_tour.tours").add("portal_project_sharing_chatter_mention_
         { trigger: "table > tbody > tr a:has(span:contains(Project Sharing))", run: "click" },
         { trigger: ".o_project_sharing" },
         { trigger: ".o_kanban_record:contains('Test Task')", run: "click" },
+        { trigger: ".o-mail-Composer-input", run: "edit @xxx" },
+        {
+            trigger: ".o-mail-Chatter-top:not(:has(.o-mail-Composer-suggestion))",
+            run: async () => {
+                await delay(odoo.loader.modules.get("@mail/core/common/suggestion_hook").DELAY_FETCH); // Wait for debounce time to be over
+            },
+        },
         { trigger: ".o-mail-Composer-input", run: "edit @Georges" },
-        { trigger: ".o-mail-Composer-suggestion:contains('Georges')" },
+        { trigger: ".o-mail-Chatter-top .o-mail-Composer-suggestion:contains('Georges')" },
```

Comment:
Why have `.o-mail-Chatter-top` in selectors? `.o-mail-Composer-suggestion` looks enough for me, and shorter is better.